cmake/prerelease.lock

```diff
@@ -0,0 +1,3 @@
+-> When this file is present, an executable that identifies as a pre-release is built by default.
```

Comment:
Do the `->` have a special meaning here?

Reply:
Yea, it doesn't. I felt presenting it as bullet points would make it more readable.
But yea, I'll remove it in the next push